docs/src/domains/steps/subdomains.md

```diff
@@ -85,7 +85,7 @@ So you could add `beep.example.com` to a different project without any issues.
 
 ## Locked domains
 
-In certain cases (such as if your domain was added manually by Platform.sh support),
+In certain cases (such as if your domain was manually added by the support team),
```

Comment:
:hammer_and_wrench: I feel like this decreases clarity because it removes the idea of what support team it is. I'm also not sure why the word order was changed, since it seems to me to be the wrong emphasis now.
```suggestion
In certain cases (such as if your domain was added manually by Platform.sh support),
```